corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-webapp/content.scss

```diff
@@ -18,12 +18,19 @@ body {
     font-size: 1.5rem;
     text-transform: uppercase;
     color: $cc-neutral-mid;
-    padding-left: 1.5rem;
+    padding-left: 12px;
```

Comment:
Is there a relative padding that could be used here instead (e.g. `1rem`)?

Reply:
There's also `$spacer`, though I'm not certain when you'd use that over `rem` (I believe `$spacer: 2rem`)

Reply:
I used px because because `.module-column-icon` does too and I did not want to mix two units.

Reply:
I don't think this needs to block, but another option would be to use a common reference variable, if the two items need to line up.  Can you explain the connection between `.page-header > h1` and `.module-column-icon`?  I see one is set to 12px and the other to 20px, is there an 8px element in the mix somewhere too?